server/channels/api4/bot_test.go

```diff
@@ -719,21 +804,28 @@ func TestGetBots(t *testing.T) {
 	})
 	require.NoError(t, err)
 	CheckCreatedStatus(t, resp)
-	defer th.App.PermanentDeleteBot(th.Context, bot3.UserId)
+	defer func() {
+		err := th.App.PermanentDeleteBot(th.Context, bot3.UserId)
+		assert.Nil(t, err)
+	}()
 
 	deletedBot2, resp, err := th.SystemAdminClient.CreateBot(context.Background(), &model.Bot{
 		Username:    GenerateTestUsername(),
 		Description: "a deleted bot",
 	})
 	require.NoError(t, err)
 	CheckCreatedStatus(t, resp)
-	defer th.App.PermanentDeleteBot(th.Context, deletedBot2.UserId)
+	defer func() {
+		err := th.App.PermanentDeleteBot(th.Context, deletedBot2.UserId)
```

Comment:
Nit: Given that `err` is of type `*model.AppError`, it would be nice to use `appErr` as a variable name.